src/web/vue.config.js

```diff
@@ -6,8 +6,9 @@ module.exports = {
     disableHostCheck: true,
     proxy: {
       "/api": {
-        target: "http://localhost:5000",
+        target: "http://yacs_api:5000",
```

Comment:
The API url will be `yacs_api` only when running the whole application in docker. When running outside of docker, this needs to be `localhost`. 

To do this, we can allow overriding the hostname of the api through an environment variable like `YACS_API_HOSTNAME` but provide a default of `"localhost"` if `YACS_API_HOSTNAME` is not specified. Then, in `docker-compose.development.yml`, set the environment variable for the `yacs_web` service.